tests/roles/development_environment/defaults/main.yaml

```diff
@@ -1,5 +1,17 @@
 prelaunch_test_instance: true
 prelaunch_test_instance_script: pre_launch.bash
+# Ping test activation flag, note that it needs
+# prelaunch_test_instance as well.
+ping_test: true
+# Relative to
+# ~/src/github.com/openstack-k8s-operators/data-plane-adoption
+# Collected in
+# /controller/data-plane-adoption-tests-repo/data-plane-adoption/
+ping_test_log_file: ./tests/logs/ping_results.log
+# max cut in seconds
+ping_test_loss_threshold: 0
+# max cut in percent
+ping_test_loss_threshold_percent: 0
```

Comment:
The default thresholds will have to be higher :). Until 17.1.4 is released, we'll have significant ping loss due to https://issues.redhat.com/browse/OSPRH-10283

Reply:
Adjusted.  Now the threshold are high enought too pass the current failure.  What will happen though, is that a warning will be registered in the Ansible output and a log file with human readable datetime will be produced.